runtimes/rust/src/lib.rs

```diff
@@ -351,16 +350,31 @@ impl RustRuntime {
             .expect("Method called before Rust runtime is initialized")
     }
 
-    /// Adds a new service factory to the runtime and returns
+    /// Adds a new available to deploy service factory to the runtime and returns
     /// a modified `RustRuntime` object for further chaining.
-    pub fn with_factory(mut self, service_factory: impl Into<Box<dyn ServiceFactory>>) -> Self {
+    pub fn with_available_service(
```

Comment:
Why did you change the name? I liked the old one better; for one, it's shorter 🙃 It also reflects the naming of the argument type (`ServiceFactory`).